### PR TITLE
Search button does not search

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
Not ready for review yet.

**Comment by smn** [comment]
search

Ready for review but still in need of a test for the search view.

**Comment by smn** [comment]
This article happens to have the word "blockquote" and it's the only result.

**Comment by bruskiza** [comment]
:+1:

File: ndohyep/templates/search/search_results.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags comments %}
+
+{% block title %}Search{% endblock %}
+
+{% block content %}
+<div class="title grey">
+	<h1>You searched for {{search_query}}</h1>
```
Comment (miltontony) [inline ndohyep/templates/search/search_results.html:8]: missing `{% trans %}`

File: ndohyep/templates/search/search_results.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags comments %}
+
+{% block title %}Search{% endblock %}
```
Comment (miltontony) [inline ndohyep/templates/search/search_results.html:4]: missing `{% trans %}`

File: ndohyep/templates/search/search_results.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            Page {{ search_results.number }} of {{ search_results.paginator.num_pages }}
+        </span>
+
+        {% if search_results.has_next %}
+            <a href="?page={{ search_results.next_page_number }}&amp;q={{search_query}}">&rarr;</a>
+        {% endif %}
+        </div>
+    </div>
+{% else %}
+    <p>No search results for {{search_query}}.</p>
```
Comment (miltontony) [inline ndohyep/templates/search/search_results.html:36]: missing `{% trans %}`

File: sideloader/postinstall.sh [hunk 4]
```diff
@@ -9,3 +9,6 @@ $manage compress
 
 # compile i18n strings
 $manage compilemessages
+
+# Update the search index
+$manage update_index
```
Comment (miltontony) [inline sideloader/postinstall.sh:14]: Hmm... It think we need to look at the cron tasks in puppet to ensure we add them.

**Comment by smn** [comment]
@miltontony landing, adds search, pls review in the morning.

**Pull request merged**